Documentation/grpc-metadata.md

```diff
@@ -223,3 +223,49 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
     stream.SetTrailer(trailer)
 }
 ```
+
+## Updating metadata in the interceptor - server side
+
+Server side metadata updating in the interceptor examples are available [here](../examples/features/metadata_interceptor/server/main.go).
```

Comment:
Please wrap these lines at 80 cols width. Thanks.

Reply:
Got it. I will fix it

Reply:
Could you please update this line to look like:

```
An example for updating metadata from a server interceptor is 
available [here](../examples/features/metadata_interceptor/server/main.go).
```

Also, please ensure that this line is also wrapped to 80-cols width. Thanks.